src/Abp.Web.Resources/Abp/Framework/scripts/libs/abp.signalr-client.js

```diff
@@ -69,7 +69,7 @@ var abp = abp || {};
 
         // Add query string: https://github.com/aspnet/SignalR/issues/680
         if (abp.signalr.qs) {
-            url += '?' + abp.signalr.qs;
+            url += (url.indexOf('?') >= 0 ? '&' : '?') + abp.signalr.qs;
```

Comment:
`== -1` is better as it's an explicit check.

Reply:
Sounds reasonable